app/frontend/src/features/communities/useCommunity.ts

```diff
@@ -0,0 +1,122 @@
+    {
+      enabled: !!communityId,
+      getNextPageParam: (lastPage) =>
+        lastPage.nextPageToken ? lastPage.nextPageToken : undefined,
+    }
+  );
+
+export const useListGroups = (communityId?: number) =>
+  useInfiniteQuery<ListGroupsRes.AsObject, GrpcError>(
+    communityGroupsKey(communityId!),
+    ({ pageParam }) => service.communities.listGroups(communityId!, pageParam),
+    {
+      enabled: !!communityId,
+      getNextPageParam: (lastPage) =>
+        lastPage.nextPageToken ? lastPage.nextPageToken : undefined,
+    }
+  );
+
+export const useListPlaces = (communityId?: number) =>
+  useInfiniteQuery<ListPlacesRes.AsObject, GrpcError>(
+    communityPlacesKey(communityId!),
+    ({ pageParam }) => service.communities.listPlaces(communityId!, pageParam),
+    {
+      enabled: !!communityId,
+      getNextPageParam: (lastPage) =>
+        lastPage.nextPageToken ? lastPage.nextPageToken : undefined,
+    }
+  );
+
+export const useListGuides = (communityId?: number) =>
+  useInfiniteQuery<ListGuidesRes.AsObject, GrpcError>(
+    communityGuidesKey(communityId!),
+    ({ pageParam }) => service.communities.listGuides(communityId!, pageParam),
+    {
+      enabled: !!communityId,
+      getNextPageParam: (lastPage) =>
+        lastPage.nextPageToken ? lastPage.nextPageToken : undefined,
+    }
+  );
+
+export const useListDiscussions = (communityId?: number) =>
+  useInfiniteQuery<ListDiscussionsRes.AsObject, GrpcError>(
+    communityDiscussionsKey(communityId!),
+    ({ pageParam }) =>
+      service.communities.listDiscussions(communityId!, pageParam),
+    {
+      enabled: !!communityId,
+      getNextPageParam: (lastPage) =>
+        lastPage.nextPageToken ? lastPage.nextPageToken : undefined,
+    }
+  );
+
+export const useListAdmins = (communityId?: number) =>
+  useInfiniteQuery<ListAdminsRes.AsObject, GrpcError>(
+    communityAdminsKey(communityId!),
+    ({ pageParam }) => service.communities.listAdmins(communityId!, pageParam),
+    {
+      enabled: !!communityId,
+      getNextPageParam: (lastPage) =>
+        lastPage.nextPageToken ? lastPage.nextPageToken : undefined,
+    }
+  );
+
+export const useListMembers = (communityId?: number) =>
+  useInfiniteQuery<ListMembersRes.AsObject, GrpcError>(
+    communityMembersKey(communityId!),
+    ({ pageParam }) => service.communities.listMembers(communityId!, pageParam),
+    {
+      enabled: !!communityId,
+      getNextPageParam: (lastPage) =>
+        lastPage.nextPageToken ? lastPage.nextPageToken : undefined,
+    }
+  );
+
+export const useListNearbyUsers = (communityId?: number) =>
+  useInfiniteQuery<ListNearbyUsersRes.AsObject, GrpcError>(
+    communityNearbyUsersKey(communityId!),
+    ({ pageParam }) =>
+      service.communities.listNearbyUsers(communityId!, pageParam),
+    {
+      enabled: !!communityId,
+      getNextPageParam: (lastPage) =>
+        lastPage.nextPageToken ? lastPage.nextPageToken : undefined,
+    }
```

Comment:
Same question as on `DiscussionCard` - considering we're only displaying single page of content on the "landing page" of a community at the moment.

Reply:
Yes I figured these will all be needed soon.